setup.py

```diff
@@ -22,7 +21,22 @@
 
 dist_name = "kolibri"
 
-readme = io.open("README.md", mode="r", encoding="utf-8").read()
+# RST-formatted long description.
+# (Markdown will not work until we upgrade build dependencies)
+readme = """
+`Kolibri <https://learningequality.org/kolibri/>`_ is the offline learning platform
+from `Learning Equality <https://learningequality.org/>`_.
+
+This package can be installed by running ``pip install kolibri``. `See the download
+page <https://learningequality.org/kolibri/>`_ for other methods of installation.
```

Comment:
I think the download page url is https://learningequality.org/download/?

Reply:
thanks @lyw07 @jonboiser